go.mod

```diff
@@ -3,13 +3,14 @@ module github.com/banzaicloud/kafka-operator
 go 1.15
 
 require (
-	emperror.dev/errors v0.7.0
+	emperror.dev/errors v0.8.0
 	github.com/Shopify/sarama v1.27.1
 	github.com/banzaicloud/bank-vaults/pkg/sdk v0.3.1
 	github.com/banzaicloud/istio-client-go v0.0.9
 	github.com/banzaicloud/istio-operator v0.0.0-20200407070503-3f7dc6953a7b
 	github.com/banzaicloud/k8s-objectmatcher v1.4.1
 	github.com/banzaicloud/kafka-operator/api v0.0.0
+	github.com/banzaicloud/kafka-operator/properties v0.0.0-20210128180953-6e7201a76903
```

Comment:
I think we should handle this as above. It means we should use api v0.0.0 and use replace which points to the dir where the properties lives.
WIth that approach we can update the properties dir without any go.mod modification.

Reply:
Agreed. Will update the PR as soon as the #543 got merged.